cypress/integration/login.js

```diff
@@ -1,122 +1,95 @@
+import LoginPage from '../obj/loginPage';
+
+const loginPage = new LoginPage();
+
 describe('Sourcebooks login', function() {
 
     it('Should display validation for empty user after attempted loggin', function () {
         
-        cy.visit('/');
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click();
-        cy.get('.Select.not-valid').should('be.visible')
+        loginPage.visit();
+        loginPage.getUserValidationIndicator().should('not.visible')
+        loginPage.getSubmitButton().click();
+        loginPage.getUserValidationIndicator().should('be.visible')
     })
 
     it('Should be able to login with role Team Lead', function () {
-
-        cy.visit('/');
-        cy.get('[id="loginForm.userId"]').click({force:true});
-        cy.get('[aria-label="Valentinas Kasteckis"]').click();
-        cy.get('[id="loginForm.role"]').click({force:true});
-        cy.get('[aria-label="Team Lead"]').click();
-        cy.get('[type="submit"]').click();
-
-        cy.url().should('include', '/time-logging');
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('Valentinas Kasteckis');
-        cy.get('.main-nav').find('li').should('have.length', 2);
+        loginPage.visit();
+        loginPage.getUserDropDown().click({force:true});
+        loginPage.getSpecificUserFromDropDown("Valentinas Kasteckis").click();

```

Comment:
As you often use same User it would be wise to extract it as a variable and then reuse it in all instances of this function and loginPage.checkLoginName("Valentinas Kasteckis");